client/js/components/statement/assessmentTable/DpMoveStatementModal.vue

```diff
@@ -23,31 +23,21 @@
         v-if="isLoading"
         class="u-pv-0_5" />
       <template v-else>
-        <div class="flash flash-warning flow-root">
-          <i class="fa fa-exclamation-triangle u-mt-0_125 float-left" />
-          <div class="u-ml">
-            <p
-              :class="{'u-mb-0': false === hasPermission('feature_statement_move_to_foreign_procedure')}"
-              :inner-html="Translator.trans('statement.moveto.procedure.description')" />
-            <p
-              class="u-mb-0"
-              v-if="hasPermission('feature_statement_move_to_foreign_procedure')"
-              :inner-html="Translator.trans('statement.moveto.procedure.description.foreignProcedures')" />
-          </div>
-        </div>
-
+        <dp-inline-notification
+          class="mt-3 mb-2"
+          :message="Translator.trans('statement.moveto.procedure.description')"
+          type="warning" />
+        <dp-inline-notification
+          v-if="hasPermission('feature_statement_move_to_foreign_procedure')"
+          class="mt-3 mb-2"
+          :message="Translator.trans('statement.moveto.procedure.description.foreignProcedures')"
+          type="warning" />
         <!-- display if user is not the assignee of all fragments of this statement or if any fragments of this statement are currently assigned to departments -->
-        <div
-          class="flash flash-warning flow-root"
-          v-if="(userIsAssigneeOfAllFragments && fragmentsAreNotAssignedToDepartments) === false">
-          <i class="fa fa-exclamation-triangle u-mt-0_125 float-left" />
-          <div class="u-ml">
-            <p
-              class="u-mb-0"
-              :inner-html="Translator.trans('statement.moveto.procedure.fragments.not.claimed.warning')" />
-          </div>
-        </div>
-
+        <dp-inline-notification
+          v-if="!userIsAssigneeOfAllFragments && !fragmentsAreNotAssignedToDepartments"
+          class="mt-3 mb-2"
```

Comment:
is it possible to use the margins only in one direction (top or bottom)? In the passt we ran in some trouble to manage the spacing for the notification state displayed/hidden with the outcome, that its Easier if all Elements bring space all for the same direction. I hope, You get my point

Reply:
yes, of course, I've copied this part and forgot to change the classes. thx for spotting this!